site/src/components/notification.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Notification
+description: The Notification component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D1002%253A7106%26t%3DWU64bU98uaT1nU7A-1
+classType: Component
+classPrefixes:
+  - notification
+---
+
+# Notification
+
+Notification is a colored block meant to alert users of something.
```

Comment:
Similar feedback to what I said on the Banner page: can we provide a paragraph that explains the difference between Notifications and Banners, and links out to the Banner component page?

Reply:
Likewise, can we provide some sample intended use cases for Notifications?

Reply:
And maybe we don't use the word `.alert`?

Reply:
Updated